tensorflow/python/ops/math_ops.py

```diff
@@ -2795,6 +2795,23 @@ def _calc_mat_mul_flops(graph, node):
   return ops.OpStats("flops", (k * output_count * 2))
 
 
+@ops.RegisterStatistics("BatchMatMul", "flops")
+def _calc_mat_mul_flops(graph, node):
+  """Calculates the compute resources needed for BatchMatMul."""
+  transpose_a = node.attr["transpose_a"].b
+  a_shape = graph_util.tensor_shape_from_node_def_name(graph, node.input[0])
+  a_shape.assert_is_fully_defined()
+  batch_size = np.prod(a_shape[:-2])
```

Comment:
few lint errors: Unused variable 'batch_size' [unused-variable]

Reply:
Hi @ry, I just removed the unused batch size, thanks for pointing that out